managing-os/templates/archlinux.md

```diff
@@ -18,367 +18,34 @@ to test new features/updates.
-    
-        *   **$ make qubes-vm**
-        <br>
-        <br>
-![arch-template-20](/attachment/wiki/ArchlinuxTemplate/arch-template-20.png)
-        <br>
-        <br>
-        
-        
-            *   These are the indivual component 'make' commands:
-    
-                *   **$ make vmm-xen-vm**
-          
-                *   **$ make core-vchan-xen-vm**
-           
-                *   **$ make core-qubesdb-vm**
-           
-                *   **$ make linux-utils-vm**
-           
-                *   **$ make core-agent-linux-vm**
-           
-                *   **$ make gui-common-vm**
-           
-                *   **$ make gui-agent-linux-vm**
-<br>
-<br>
-
-
-##### **10:   Make the actual Archlinux template:** #####
-
-*   **$ make template**  
-<br>
-<br>
-![arch-template-21](/attachment/wiki/ArchlinuxTemplate/arch-template-21.png)
-<br>
-<br>
-<br>
-
-
-##### **11:   Transfer Template into Dom0** #####
-
-*   You need to ensure these two files are in the '**noarch**' directory 
+    pub   2048R/2043E7ACC1833B9C 2014-03-27 [expires: 2018-03-29]
+          Key fingerprint = D85E E12F 9678 51CC F433  515A 2043 E7AC C183 3B9C
+    uid                          Olivier MEDOC (Qubes-OS signing key) <o_medoc@yahoo.fr>
 
-    *   **$ cd /home/user/qubes-builder/qubes-src/linux-template-builder/rpm/**
+The key is not installed when you install (signed) template package. You can however choose to trust it by registering it into pacman:
 
-    *   **$ ls**   *(confirm the below two files are there)*
+    # pacman-key --recv-key 2043E7ACC1833B9C
+    # pacman-key --finger 2043E7ACC1833B9C
+ 
+If the fingerprint is correct, you can then sign the key:
 
-        *   **install-templates.sh**   (script to install template in dom0)
+    # pacman-key --lsign-key 2043E7ACC1833B9C
 
-    *   **$ cd noarch**
-
-    *   **$ ls**
-
-        *   **qubes-template-archlinux-X.X.X-XXXXXXXXXXXX.noarch.rpm**  (this is the template package 'X' replaces version and build digits)
-<br>
-<br>
-![arch-template-22](/attachment/wiki/ArchlinuxTemplate/arch-template-22.png)
 <br>
-<br>
-
-*   **Transfer the install-templates.sh script file into Dom0**
-  *Note: as there is not a typical file transfer method for Dom0, for security reasons, this less than simple transfer function has to be used*
-
-    *   Swtich to Domo and open a terminal window.
 
-    **Note:** Take care when entering these cmd strings.  They are very long and have a number of characters that are easy to mix '**-**' vs '**.**' '**<u>T</u>emplates** (correct) vs **<u>t</u>emplates** (wrong) or **Template_**'(also wrong)  This script will also take care of transfering the actual template.rpm to Dom0 as well.
+Install
+-------
 
+A prebuilt template is available only for Qubes 3.2. Before Qubes 3.2, it should be compiled from source as described in [building-archlinux-template](/doc/building-archlinux-template/).
```

Comment:
I think this page is somehow outdated. Maybe better to replace it with those parts you're removing from here?